### ISSUE TITLE
Support batched list items

### ISSUE THREAD

**Issue by jpsim** [realm/jazzy#388]
I just learned that this is valid documentation syntax:

``` swift
/// docs
///
/// - Parameters:
///   - first: this is void
///   - second: this is also void
public func a(first: Void, second: Void)
```

Xcode picks that up just like the `- parameter first: ...` syntax.

**Comment by segiddins** [comment]
This shouldn't be too too hard to parse -- just taking the nested list off of the parameters list item

**Comment by pcantrell** [comment]
It seems like it might be time (soon) to take another look at the parsed XML emitted by SourceKit. It could bring Jazzy into full syntax compliance in one fell swoop.

**Comment by jpsim** [comment]
> It seems like it might be time (soon) to take another look at the parsed XML emitted by SourceKit. It could bring Jazzy into full syntax compliance in one fell swoop.

I wouldn't want to dive too deep into that given that Swift's open sourcing is scheduled to happen within the next month-ish at which point we may want to take a different approach altogether.

**Comment by pcantrell** [comment]
> Swift's open sourcing is scheduled to happen within the next month-ish

Good point.

**Comment by jpsim** [comment]
We should support this batched item syntax for all special list items (SeeAlso was requested in #384).

**Comment by aamctustwo** [comment]
This batched format is now the default when using Xcode 8.1's automatic generation of documentation comments.

**Comment by istx25** [comment]
Seems to me that this issue has been resolved. Closing.

**Comment by istx25** [comment]
Actually, I may have misread. Feel free to re-open if that was the case. My apologies if it is!

**Comment by aamctustwo** [comment]
@istx25 Nope, still an issue. :smile_cat: I was just commenting that that since this is now a default format for Xcode's built in documentation comment generator, it's going to be even more common place.

**Comment by istx25** [comment]
haha thought so, thanks @aamctustwo!

**Comment by johnfairh** [comment]
Closing, this is now at parity with Xcode.
